pkg/webui/console/components/rights-group/index.js

```diff
@@ -186,12 +190,17 @@ class RightsGroup extends React.Component {
 
   @bind
   handleGrantTypeChange(val) {
-    const { onChange, pseudoRight } = this.props
+    const { onChange, pseudoRight, derivedRights } = this.props
     const { individualRightValue } = this.state
 
     if (val === 'pseudo') {
       onChange([pseudoRight])
+    } else if (val === 'readonly') {
+      this.setState({ selectReadOnly: true })
+      const readOnlyRights = derivedRights.filter(right => right.includes('_READ'))
```

Comment:
It's not only the rights that contain the word `READ`, unfortunately. It's necessary to go through the list and add rights individually. In that regard, does it make sense to add a new pseudo right for read only rights, @htdvisser? Otherwise we'll have to maintain a list in the console and update it any time a new read-only right is added.

If we don't add a pseudo right, we should add a (maintained) constant module that contains the read only rights and then use this list to intersect it with the available rights that can be granted within the rights-group component (like you currently do with the filter).

Reply:
I don't think we should add a pseudo right for "read only" rights, because in most cases that isn't actually what you want (if you don't trust your friend enough to give them write access, you probably also don't want them to read your device keys).